Formula/hcl2json.rb

```diff
@@ -0,0 +1,33 @@
+class Hcl2json < Formula
+  desc "Convert hcl2 to json"
```

Comment:
```suggestion
  desc "Convert HCL to JSON"
```
or
```suggestion
  desc "Convert HCL2 to JSON"
```
(I'm assuming HCL2 is just the next major version of HCL.)